ts/Core/Defaults.ts

```diff
@@ -126,6 +126,19 @@ const defaultOptions: DefaultOptions = {
      * ```
      */
     lang: {
+        stockOpen: 'Open',
+        stockHigh: 'High',
+        stockLow: 'Low',
+        stockClose: 'Close',
```

Comment:
Those are specific to Stock and shouldn't weigh down the core.

Reply:
See https://github.com/highcharts/highcharts/blob/4e64984bbde6fe6c825708fe57d6d98f1e8fb59e/ts/Stock/RangeSelector/RangeSelectorDefaults.ts#L44 for an example of extending `lang` in a module.

Reply:
Still valid as of January 20 - these four keys should be removed from the core.